locale/es_ES/locale.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<locale name="es_ES" full_name="Español (España)">
+	
+	<!-- Theme description -->
+	<message key="plugins.themes.pragma.name">Tema Theme</message>
+	<message key="plugins.themes.pragma.description">El tema para OJS 3</message>
+	
+	<message key="plugins.themes.pragma.more-info">Más información</message>
+	<message key="plugins.themes.pragma.issues.recent">Ediciones recientemente publicadas</message>
+	<message key="plugins.themes.pragma.issue.published">Publicado</message>
+	<message key="plugins.themes.pragma.issue.description">Descripción de la publicación</message>
+	<message key="plugins.themes.pragma.registration.consent">Consentimiento</message>
+	<message key="plugins.themes.pragma.mainMenu">Menú principal</message>
+	<message key="plugins.themes.pragma.language.toggle">Cambiar el idioma. El idioma actual es:</message>
+	<message key="plugins.themes.pragma.adminMenu">Menú Administración</message>
+	<message key="plugins.themes.pragma.issue.fullIssueLink">Ver el número completo</message>
+	<message key="plugins.themes.pragma.article.biography">Biografía</message>
+	<message key="plugins.themes.pragma.article.figure">Imagen de portada</message>
+	<message key="plugins.themes.pragma.authors.browse">Navegar por autor</message>
```

Comment:
The Spanish translation of OJS uses `autor/a`, maybe change here too? (And on next line)